### PR TITLE
move db checkpoints modules into their own namespace

### PR DESCRIPTION
**Pull request by paolino**
- I have moved CheckpointsOld Checkpoints and Addressbook into Checkpoints namespace

### Issue Number

ADP-1784

### CODE REVIEW

File: lib/core/cardano-wallet-core.cabal [hunk 1]
```diff
@@ -195,11 +195,11 @@ library
...
       Cardano.Wallet.Compat
       Cardano.Wallet.DB
-      Cardano.Wallet.DB.Checkpoints
+      Cardano.Wallet.DB.Checkpoints.Model
       Cardano.Wallet.DB.MVar
       Cardano.Wallet.DB.Model
-      Cardano.Wallet.DB.Sqlite.AddressBook
-      Cardano.Wallet.DB.Sqlite.CheckpointsOld
+      Cardano.Wallet.DB.Checkpoints.AddressBook
+      Cardano.Wallet.DB.Checkpoints.Store
```
Comment (HeinrichApfelmus) [inline lib/core/cardano-wallet-core.cabal]: Alphabetical order in .cabal file?

**Review by HeinrichApfelmus — APPROVED** [review]
Very pleasant to review! 😊 I actually want to move two of these files again, specifically

* `Cardano.Wallet.DB.Checkpoints.Model` → `Cardano.Wallet.Checkpoints`
* `Cardano.Wallet.DB.Checkpoints.AddressBook` → `Cardano.Wallet.Address.Book`

But happy to merge as is (but see comment) so that you can move forward.

**Review by HeinrichApfelmus — APPROVED** [review]

**Comment by paolino** [comment]
bors try

**Pull request merged**